### ISSUE TITLE
cURL error 35: SSL connect error

### ISSUE THREAD

**Issue by zekylaf** [seegno/uphold-sdk-php#130]
Hi, I need help please! I am getting this error and I don't know how to solve it. Thanks

in ErrorHandler.php line 49
at ErrorHandler->onException(object(ConnectException)) in HttpClient.php line 142
at HttpClient->request('v0/me', null, 'GET', array('Accept' => 'application/json', 'Content-Type' => 'application/json', 'User-Agent' => 'uphold-sdk-php v4.0.3 (https://github.com/seegno/uphold-sdk-php)', 'Authorization' => 'Bearer <token>'), array('query' => array())) in HttpClient.php line 87
at HttpClient->get('v0/me', array(), array('Accept' => 'application/json', 'Content-Type' => 'application/json', 'User-Agent' => 'uphold-sdk-php v4.0.3 (https://github.com/seegno/uphold-sdk-php)', 'Authorization' => 'Bearer <token>')) in UpholdClient.php line 313
at UpholdClient->get('/me') in UpholdClient.php line 228

**Comment by nunorafaelrocha** [comment]
@zekylaf Attention! Don't publish your personal access token! People can access your Uphold account with that. I've edit your comment and remove the token, but someone may already have stolen your access token.

You need to revoke it!

About your question, that seems a connectivity problem. Have you check this issue #124?

**Comment by zekylaf** [comment]
Thanks for answer @nunorafaelrocha! And thanks for edit my post too. I am going to revoke the access token

I saw yesterday the issue that you are telling me. I make all the steps that your link says, but the problem persist.

I want to ask you if you know any way to disable the SSL check from uphold-sdk. I think that this is a Guzzle problem. 

Please, I need to solve this, thanks you

**Comment by nunorafaelrocha** [comment]
@zekylaf First I would recommend you try a curl request from your terminal to ensure connection to Uphold's API using this command:

```sh
curl "https://api.uphold.com/v0/me" \
  -H "Authorization: Bearer <token>"
```

Second, I would verify the PHP configuration as described in [this article](http://snippets.webaware.com.au/howto/stop-turning-off-curlopt_ssl_verifypeer-and-fix-your-php-config/).

**Comment by zekylaf** [comment]
Hi @nunorafaelrocha. I solved the problem. I have an old version called "Bitreserve-sdk-php" that I have used time ago to connect my app with Uphold and I never have problems with this version, so I replace some files of the "4.0.3" version with those of the older version. Then I have changed the files'  namespaces.

Finally, it starts working, without errors. If you like, If you like, I can upload all.

Thanks for the help
